app/models/host.rb

```diff
@@ -21,6 +21,7 @@ class Host < ApplicationRecord
     "kubevirt"        => "KubeVirt",
     "vmware"          => "VMware",
     "openstack_infra" => "OpenStack Infrastructure",
+    "ibm_power_hmc"   => "IBM Power HMC",
```

Comment:
Was going to suggest moving this to be alphabetic but this whole list is wrong already :laughing: